text_practice.rb

```diff
@@ -0,0 +1,15 @@
+require 'rubygems' # not necessary with ruby 1.9 but included for completeness
+require 'twilio-ruby'
+
+# put your own credentials here
+account_sid = 'ACd18bfc872e98296bfd513c3537d79945'
+auth_token = 'bb86f5ef30b2e093ec06255279f7ece8'
+
+# set up a client to talk to the Twilio REST API
+@client = Twilio::REST::Client.new account_sid, auth_token
+
+@client.account.messages.create({
+  :from => '+441698313072',
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Use 2 spaces for indentation in a hash, relative to the first position after the preceding left parenthesis.